app/templates/index.html

```diff
@@ -89,6 +89,19 @@
 <div class="grid-row">
     <div class="column-two-thirds">
 
+  {% if current_user.has_role('admin') %}
```

Comment:
Do we have any tests to assert what each user role can view on the Admin home page?

Reply:
Ooh probably yes - I should add one for this.  I'd forgotten about the homepage link and only added it at the end of Friday after doing all the other tests.

Reply:
Hmm, turns out we don't have a lot around this. We do have functional tests that check this stuff, but feels like it should be unit tested.  I'll add some for my new bits at least.